src/main/java/org/apache/solr/handler/dataimport/DocBuilder.java

```diff
@@ -114,8 +114,10 @@ private VariableResolver getVariableResolver() {
       VariableResolver resolver = null;
       String epoch = propWriter.convertDateToString(EPOCH);
       if(dataImporter != null && dataImporter.getCore() != null
-          && dataImporter.getCore().getResourceLoader().getCoreProperties() != null){
-        resolver =  new VariableResolver(dataImporter.getCore().getResourceLoader().getCoreProperties());
+              && dataImporter.getCore().getCoreContainer() != null
+              && dataImporter.getCore().getCoreContainer().getConfig() != null
+              && dataImporter.getCore().getCoreContainer().getConfig().getSolrProperties() != null) {
```

Comment:
note: I am not 100% sure if these are correct properties. I found properties in a few different places.